src/index.tsx

```diff
@@ -78,13 +79,15 @@ export const App = () => {
                                 <MobileTokenContextProvider>
                                   <AppLanguageProvider>
                                     <GeolocationContextProvider>
-                                      <GlobalMessagesContextProvider>
-                                        <BottomSheetProvider>
-                                          <FeedbackQuestionsProvider>
-                                            <RootStack />
-                                          </FeedbackQuestionsProvider>
-                                        </BottomSheetProvider>
-                                      </GlobalMessagesContextProvider>
+                                      <AnalyticsContextProvider>
```

Comment:
Is it intentional that the `AnalyticsContextProvider` is added both here and under `NavigationContainer`?

Reply:
Yes, It's necessary to get auto capture to work. Auto capture must be set up with navigation available.